src/pages/studyView/styles.module.scss

```diff
@@ -18,4 +18,14 @@
 
 .studyViewSurvivalPlotBody {
     position: relative
+}
+
+.studyViewAutoMargin {
```

Comment:
not too important, but when you are using css modules, you don't have to prefix your class names because they are automatically made unique.